pkg/jx/cmd/step_create_task.go

```diff
@@ -169,6 +169,9 @@ func (o *StepCreateTaskOptions) Run() error {
 	}
 	o.devNamespace = ns
 
+	if o.Verbose {
```

Comment:
@jstrachan I think, we should configure the `logrus` log-level to `info` in one place before starting the command execution when `Verbose` flag is set. This will have the benefit that we won't need to guard the logs with `if o.Verbose {}`.